utilities/_utilities.clears.scss

```diff
@@ -0,0 +1,15 @@
+.u-clearfix {
```

Comment:
Why did we need this done as part of this branch?

I'm thinking this be ported into another branch todo for the future.

Reply:
ok have transferred this to another branch for now